app/controllers/AccountManagement.scala

```diff
@@ -132,6 +133,21 @@ object ManageDelivery extends ContextLogging {
     }
   }
 
+  def fulfilmentCheck(trackDelivery: TrackDelivery)(implicit request: Request[TrackDelivery], touchpoint: TouchpointBackend.Resolution): Future[Result] = {
+    implicit val tpBackend = touchpoint.backend
+    logger.info(s"Attempting to perform tracking lookup: $trackDelivery")
+    val futureLookupAttempt = FulfilmentLookupService.lookupSubscription(tpBackend.environmentName, trackDelivery)
+    futureLookupAttempt.map { lookupAttempt => lookupAttempt match {
+        case \/-(lookup) =>
+          logger.info(s"Successful delivery tracking for $trackDelivery; showing deliveryTrackingSuccess")
+          Ok(views.html.account.deliveryTrackingSuccess(lookup, trackDelivery.issueDate))
+        case -\/(_) =>
```

Comment:
so here, keep hold of the message in the left, and error log at this point (as this is the primary entry point/controller